Packs/Microsoft365Defender/Playbooks/playbook-Microsoft_365_Defender_-_Get_Email_URL_Clicks.yml

```diff
@@ -622,4 +627,4 @@ outputs:
   type: unknown
 tests:
 - No tests (auto formatted)
-fromversion: 6.5.0
```

Comment:
Since `Emails Indicators Hunt` is from 6.2.0, this one must also be 6.2.0

Reply:
This is a mistake, the new playbook should be from 6.8, will fix that.